test/suites/auth.sh

```diff
@@ -99,6 +99,11 @@ test_authorization() {
   LXD_CONF="${LXD_CONF3}" gen_cert_and_key "client"
   ! LXD_CONF="${LXD_CONF3}" lxc remote add tls "${tls_identity_token2}" || false
 
+  # Check token prune task works
+  [ "$(lxc auth identity list --format csv | grep -cF 'pending')" = 1 ]
```

Comment:
Please can you also add a test for checking the classic tokens are getting removed too now we have the /internal endpoint to trigger it?